packages/eslint-plugin-office-addins/src/rules/call-sync-before-read.ts

```diff
@@ -115,7 +53,7 @@ export = {
       Program(
         programNode: TSESTree.Node /* eslint-disable-line no-unused-vars */
       ) {
-        findReferences(context.getScope());
+        apiReferences = findReferences(context.getScope());
```

Comment:
Make sure this works well.  I was trying to do something link this when I was developing this rule and it wasn't working (array was getting reset or emptied unexpectedly).

Reply:
Not sure what exactly was the problem you were having, but this code structure is working fine now